rusk-profile/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rusk-profile"
-version = "0.5.0-rc.0"
+version = "0.5.1-rc.0"
```

Comment:
I think this could be `0.5.0-rc.1` but I assume we don't have any other consumer rather than rusk itself so it wouldn't mind leave as is.